lib/compress/zstd_compress.c

```diff
@@ -1003,6 +1159,12 @@ static void ZSTD_reduceIndex (ZSTD_CCtx* zc, const U32 reducerValue)
 
     { U32 const h3Size = (zc->hashLog3) ? 1 << zc->hashLog3 : 0;
       ZSTD_reduceTable(zc->hashTable3, h3Size, reducerValue); }
+
+    { if (zc->appliedParams.ldmParams.enableLdm) {
+          U32 const ldmHSize = 1 << zc->appliedParams.ldmParams.hashLog;
```

Comment:
nit: `(U32)1`, its a good habit to cast the `1` to whatever type you need. In C++ I write `auto const x = int_type(1) << y`.